core/sql/regress/hive/EXPECTED021

```diff
@@ -0,0 +1,132 @@
+Total Rows: 100000
+Total UEC:  70
+Low Value:  (1924)
+High Value: (1992)
+Intervals:  70
+
+Number    Rowcount         UEC Boundary
+====== =========== =========== ======================================
+     0           0           0 (1924)
+     1        1300           1 (1924)
+     2        1750           1 (1925)
+     3        1600           1 (1926)
+     4        1400           1 (1927)
+     5        1250           1 (1928)
+     6        1150           1 (1929)
+     7        1400           1 (1930)
+     8         800           1 (1931)
+     9        1700           1 (1932)
+    10        1300           1 (1933)
+    11        1500           1 (1934)
+    12        1550           1 (1935)
+    13        1400           1 (1936)
+    14        1700           1 (1937)
+    15        1300           1 (1938)
+    16        1800           1 (1939)
+    17        1450           1 (1940)
+    18        1450           1 (1941)
+    19        1700           1 (1942)
+    20        1750           1 (1943)
+    21        1450           1 (1944)
+    22        1150           1 (1945)
+    23        1000           1 (1946)
+    24        1250           1 (1947)
+    25        1600           1 (1948)
+    26        1550           1 (1949)
+    27        1650           1 (1950)
+    28        1550           1 (1951)
+    29        1050           1 (1952)
+    30        1500           1 (1953)
+    31        1200           1 (1954)
+    32        1700           1 (1955)
+    33        1450           1 (1956)
+    34        1600           1 (1957)
+    35        1550           1 (1958)
+    36        1100           1 (1959)
+    37        1500           1 (1960)
+    38        1750           1 (1961)
+    39         950           1 (1962)
+    40        1700           1 (1963)
+    41        1500           1 (1964)
+    42        1100           1 (1965)
+    43        1350           1 (1966)
+    44        1350           1 (1967)
+    45        1400           1 (1968)
+    46        1300           1 (1969)
+    47        1700           1 (1970)
+    48        2000           1 (1971)
+    49        1300           1 (1972)
+    50        1650           1 (1973)
+    51         850           1 (1974)
+    52        1050           1 (1975)
+    53        1300           1 (1976)
+    54        1000           1 (1977)
+    55        1350           1 (1978)
+    56        1700           1 (1979)
+    57        1100           1 (1980)
+    58        1650           1 (1981)
+    59        1700           1 (1982)
+    60        1150           1 (1983)
+    61        1100           1 (1984)
+    62        1850           1 (1985)
+    63         950           1 (1986)
+    64        1350           1 (1987)
+    65        1850           1 (1988)
+    66         950           1 (1989)
+    67        1400           1 (1990)
+    68        1250           1 (1991)
+    69        1100           1 (1992)
+    70        3200           1 (NULL)
+
```

Comment:
The answer is no. That is the reason that showstats and explain output are
excluded from the test.

On Mon, Sep 21, 2015 at 10:30 AM, Suresh Subbiah <notifications@github.com>
wrote:

> In core/sql/regress/hive/EXPECTED021
> <https://github.com/apache/incubator-trafodion/pull/87#discussion_r39984765>
> :
>
> > +    56        1700           1 (1979)
> > +    57        1100           1 (1980)
> > +    58        1650           1 (1981)
> > +    59        1700           1 (1982)
> > +    60        1150           1 (1983)
> > +    61        1100           1 (1984)
> > +    62        1850           1 (1985)
> > +    63         950           1 (1986)
> > +    64        1350           1 (1987)
> > +    65        1850           1 (1988)
> > +    66         950           1 (1989)
> > +    67        1400           1 (1990)
> > +    68        1250           1 (1991)
> > +    69        1100           1 (1992)
> > +    70        3200           1 (NULL)
> > +
>
> With sampling can we get the same results every time for a consistent test?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/87/files#r39984765>.
>



-- 
Regards, --Qifan